### PR TITLE
Edit ACNL game.dat and Wiki Entry to incorporate new pull request #3429 (fs_user: stub secure value functions)

### PR DESCRIPTION
**Pull request by robzombie91**
As of Canary d8fde9b Animal Crossing New Leaf loads game saves without corruption (Unpatched Version of ACNL & 1.5 Welcome Amiibo Update)

### CODE REVIEW

**Comment by robzombie91** [comment]
Going to test-case other versions of ACNL to see if they are also corrected by https://github.com/citra-emu/citra/commit/775fb388c29530a9e552da83b7f2ab75e9227e28

**Comment by valentinvanelslande** [comment]
Change #2647 to #3438

**Comment by robzombie91** [comment]
Doing a testcase on v1.4. Will update when testing is done

**Comment by robzombie91** [comment]
Saving works on 1.5 Welcome Amiibo Patch

File: games/animal-crossing-new-leaf/game.dat [hunk 1]
```diff
@@ -1,7 +1,7 @@
 title = "Animal Crossing: New Leaf"
 description = "Animal Crossing: New Leaf is a life simulation video game developed and published by Nintendo for the Nintendo 3DS console. In the game, the player character takes on the role of mayor in a town populated with anthropomorphic animals."
-incomplete = true
-github_issues = [2647, 2477]
+incomplete = false
```
Comment (Leo626) [inline games/animal-crossing-new-leaf/game.dat]: This line could be removed now.

**Comment by robzombie91** [comment]
Changes Made

**Pull request merged**